src/app/organization/offices/view-offices/notarios-tab/notarios-tab.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
```

Comment:
@spider0061 data tables are to be dynamically created like in clients view and not statically typed like this one.